In [2]:
# WEB DEV TEAM PLEASE DELETE BEFORE PUBLISHING - DS TEAM PLEASE KEEP IN THE FINAL VERSION OF THE USE CASE

from IPython.core.display import HTML
HTML("""
<style>
@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@600;700&family=Inter:wght@400;500&display=swap');
body,p,li,div{font-family:'Inter',sans-serif}
body{counter-reset:h2counter}
.usecase-header-bar,.usecase-title,.usecase-section-heading,.usecase-sub-section-heading,.usecase-sub-sub-section-heading,.usecase-unnumbered-heading{font-family:'Poppins',sans-serif}
.usecase-header-bar{padding:16px 20px;background-color:#059669;color:#fff;border-radius:6px 6px 0 0}
.usecase-title{font-size:1.9em;font-weight:800}
.usecase-details{padding-left:20px;padding-bottom:8px;padding-top:8px;font-size:1.1em;color:#121212}
.usecase-section-heading,.usecase-unnumbered-heading{font-weight:700;font-size:1.35em;color:#059669;padding:14px 4px 6px 4px;border-bottom:3px solid #2DBE6C;margin-top:10px}
.usecase-section-heading::before{content:none!important}
.usecase-sub-section-heading{font-weight:600;font-size:1.15em;color:#059669;margin-top:8px}
.usecase-sub-section-heading::before{content:none!important}
.usecase-sub-sub-section-heading{font-weight:500;font-size:1.05em;color:#059669;margin-top:6px}
.usecase-sub-sub-section-heading::before{content:none!important}
.usecase-body{font-size:1em;color:#121212;line-height:1.6;margin-bottom:8px}
.usecase-list{line-height:1.7}
.usecase-list ul,.usecase-list ol{padding-left:24px;margin-top:8px;margin-bottom:12px}
.usecase-list li{margin-bottom:6px}
.usecase-datasets{background-color:rgba(5,150,105,.06);border-left:4px solid #059669;padding:12px 20px;border-radius:0 4px 4px 0;margin-top:12px}
.usecase-datasets ul{margin-top:8px;margin-bottom:8px}
.usecase-contents a,.usecase-datasets a{color:#059669;text-decoration:none}
.usecase-datasets a{font-weight:500}
.usecase-contents a:hover,.usecase-datasets a:hover{text-decoration:underline}
.usecase-contents{line-height:1.9;padding-left:20px}
</style>
""")

<div class="usecase-title">Mental Wellbeing</div>

<div class="usecase-details"><b>Authored by: </b>Harley Ngo</div>
<div class="usecase-details"><b>Date Created: </b>March 2023, updated August 2026</div>
<div class="usecase-details"><b>Time to Complete:</b> 90 mins</div>
<div class="usecase-details"><b>Level: </b>Intermediate</div>
<div class="usecase-details"><b>Pre-requisite Skills: </b>Python</div>

<div class="usecase-unnumbered-heading"><b>Scenario</b></div>

<div class="usecase-body">
<i>"As a member of the city council, I want to review indicators of quality of life and city services by year, so I can prioritise funding and resources for initiatives that will enhance residents' living standards."</i>

<i>"As a resident of Future Melbourne, I want the city administration to use data-driven insights from indicators of wellbeing and quality of life to implement policies and services that address the needs of diverse communities and improve overall liveability."</i>
</div>

<div class="usecase-unnumbered-heading"><b>What this usecase will teach you</b></div>

<div class="usecase-body">By working through this notebook, you will learn how to:</div>

<div class="usecase-list">

- Download and combine multiple datasets from the Melbourne Open Data Platform API
- Fit and validate an ARIMA time-series model to forecast future values from a short yearly indicator series, including checking forecast accuracy with MAE, RMSE, MAPE, AIC and BIC
- Build an interactive chart using ipywidgets to let a user select and forecast any indicator on demand
- Filter and compare wellbeing and quality-of-life indicators across themes (economy, education, health, safety, transportation and more) for two geographic regions
- Translate comparative indicator data into evidence-based policy recommendations for a city council audience

</div>

<div class="usecase-unnumbered-heading"><b>Background</b></div>

<div class="usecase-body">The 'Mental Wellbeing' use case brings together two related data collections gathered over a 3-week investigation period: the Future Melbourne indicators of wellbeing, and indicators of quality of life and city services, both broken down by year for Greater Melbourne and the Melbourne municipality. The objective is to analyse the collected data, generate insights, and derive meaningful conclusions the city council can use to guide future interventions across themes such as the economy, education, health, safety, housing and the environment.</div>

<div class="usecase-unnumbered-heading"><b>Data Sets</b></div>

<div class="usecase-body">This use case combines two datasets from the Melbourne Open Data Platform.</div>

<div class="usecase-datasets">

- Indicators of wellbeing by year (Future Melbourne) — yearly wellbeing indicators for Future Melbourne, used for the ARIMA forecasting section. <a href="https://data.melbourne.vic.gov.au/explore/dataset/indicators-of-wellbeing-by-year-future-melbourne/table/" target="_blank">Link to Dataset</a>
- Indicators of quality of life and city services by year — yearly quality-of-life and city-service indicators for Greater Melbourne and the Melbourne municipality, grouped by theme. <a href="https://data.melbourne.vic.gov.au/explore/dataset/indicators-of-quality-of-life-and-city-services-by-year/table/" target="_blank">Link to Dataset</a>

</div>

<div class="usecase-unnumbered-heading"><b>Contents</b></div>

<div class="usecase-contents">

1. <a href="#section-1">Wellbeing Indicators Forecast</a>
2. <a href="#section-2">Economy</a>
3. <a href="#section-3">Education</a>
4. <a href="#section-4">Energy</a>
5. <a href="#section-5">Environment</a>
6. <a href="#section-6">Finance</a>
7. <a href="#section-7">Fire and Emergency Response</a>
8. <a href="#section-8">Governance</a>
9. <a href="#section-9">Health</a>
10. <a href="#section-10">Recreation</a>
11. <a href="#section-11">Safety</a>
12. <a href="#section-12">Shelter</a>
13. <a href="#section-13">Solid Waste</a>
14. <a href="#section-14">Telecommunication and Innovation</a>
15. <a href="#section-15">Transportation</a>
16. <a href="#section-16">Urban Planning</a>
17. <a href="#section-17">Wastewater</a>
18. <a href="#section-18">Water and Sanitation</a>

</div>

<div class="usecase-code-divider">The code and analysis for each section listed above begins below.</div>

In [9]:
#Import core libraries
import requests
import pandas as pd
import numpy as np
import os

import matplotlib.pyplot as plt
from statsmodels.tsa.arima.model import ARIMA
from sklearn.metrics import mean_absolute_error, mean_squared_error
from ipywidgets import interact, widgets
from IPython.display import display
from io import StringIO
from tqdm import tqdm

In [11]:
# Download datasets
def download_dataset(dataset_id, base_url='https://data.melbourne.vic.gov.au/api/explore/v2.1/catalog/datasets/'):
    format = 'csv'
    url = f'{base_url}{dataset_id}/exports/{format}'
    params = {
        'select': '*',
        'limit': -1,  # all records
        'lang': 'en',
        'timezone': 'UTC',
        'api_key': os.environ.get('MELB_DATA_API_KEY', '')
    }

    with requests.get(url, params=params, stream=True) as response:
        if response.status_code == 200:
            total_size = int(response.headers.get('content-length', 0))
            chunk_size = 1024  # 1KB per chunk
            progress_bar = tqdm(total=total_size, unit='iB', unit_scale=True, desc=f"Downloading {dataset_id}")

            content = bytearray()
            for chunk in response.iter_content(chunk_size=chunk_size):
                if chunk:  # filter out keep-alive new chunks
                    content.extend(chunk)
                    progress_bar.update(len(chunk))

            progress_bar.close()
            data = pd.read_csv(StringIO(content.decode('utf-8')), delimiter=';')
            return data
        else:
            print(f'Request failed with status code {response.status_code}')
            return None

# Dataset IDs
dataset_ids = [
    'indicators-of-wellbeing-by-year-future-melbourne',
    'indicators-of-quality-of-life-and-city-services-by-year'
]

# Initialise dictionary to hold the datasets
datasets = {}

# Download each dataset with a progress bar
for dataset_id in dataset_ids:
    datasets[dataset_id] = download_dataset(dataset_id)
    if datasets[dataset_id] is not None:
        print(f"{dataset_id} downloaded successfully.")


indicators-of-wellbeing-by-year-future-melbourne downloaded successfully.


indicators-of-quality-of-life-and-city-services-by-year downloaded successfully.


<div class="usecase-section-heading" id="section-1"><b>1. Wellbeing Indicators Forecast</b></div>

<div class="usecase-body">This section uses the Future Melbourne wellbeing dataset to forecast future values for each indicator based on its historical trend.</div>

In [15]:
wellbeing = datasets['indicators-of-wellbeing-by-year-future-melbourne']
wellbeing.head()

,indicator_label,indicator_description,unit,2012,2013,2014,2015,2016,population,source
0,Quality of life,Personal subjective wellbeing Index (average o...,%,75.6,76.7,76.8,76.9,76.5,Resident,Future Melbourne Social Survey
1,How safe you feel,How satisfied people feel with their safety (a...,%,80.3,80.8,80.4,82.0,79.6,Resident,Future Melbourne Social Survey
2,Community connectedness (18-35 yrs),How satisfied people are with feeling part of ...,%,65.4,64.5,69.1,68.1,68.0,Resident,Future Melbourne Social Survey
3,Self-reported health age standardised,People Self-Reporting Health as Excellent or V...,%,58.5,67.4,60.6,58.3,56.2,Resident,Future Melbourne Social Survey
4,Perceptions of safety - daytime,People Who Feel Safe or Very Safe in public sp...,%,NaN,NaN,96.3,97.1,90.3,City users,Council Plan Intercept Social Survey


### About the forecasting model

This section uses ARIMA (AutoRegressive Integrated Moving Average) to forecast future values for each wellbeing indicator based on its historical trend. ARIMA combines three components:
- **AR (AutoRegressive)** - uses past values to predict future ones
- **I (Integrated)** - differencing the data to make it stationary
- **MA (Moving Average)** - uses past forecast errors to improve predictions

The order `(5,1,0)` used below means: 5 autoregressive terms, 1 differencing step, 0 moving average terms. This order was chosen as a general-purpose starting point for short, yearly indicator series; forecast accuracy metrics (MAE, RMSE, MAPE, AIC, BIC) are printed alongside each forecast so the fit quality can be assessed per indicator.

In [18]:
def plot_indicator(indicator_label):
    indicator_data = wellbeing[wellbeing['indicator_label'] == indicator_label]

    year_cols = [c for c in indicator_data.columns if str(c).isdigit() and len(str(c)) == 4]
    years = indicator_data[year_cols].columns.astype(int).values
    values = indicator_data[year_cols].iloc[0].values.astype(float)

    if len(values) == 0 or np.isnan(values).any():
        print(f"Skipping {indicator_label}: insufficient or missing data")
        return

    # Cap the AR order dynamically so it never exceeds what the series length can support
    max_ar_order = max(1, min(5, len(values) - 2))

    try:
        model = ARIMA(values, order=(max_ar_order, 1, 0))
        model_fit = model.fit()
    except Exception as e:
        print(f"Skipping {indicator_label}: ARIMA failed to fit ({e})")
        return

    fitted_values = model_fit.fittedvalues
    mae = mean_absolute_error(values[1:], fitted_values[1:])
    rmse = np.sqrt(mean_squared_error(values[1:], fitted_values[1:]))
    mape = np.mean(np.abs((values[1:] - fitted_values[1:]) / values[1:])) * 100
    print(f"MAE: {mae:.2f} | RMSE: {rmse:.2f} | MAPE: {mape:.2f}% | AIC: {model_fit.aic:.2f} | BIC: {model_fit.bic:.2f}")

    future_years = np.arange(years[-1] + 1, years[-1] + 6)
    future_values = model_fit.forecast(steps=5)

    plt.figure(figsize=(10, 6))
    plt.plot(years, values, marker='o', label='Actual Data')
    plt.plot(future_years, future_values, linestyle='--', marker='o', color='r', label='Forecast')
    plt.title(f'Trend for {indicator_label}')
    plt.xlabel('Year')
    plt.ylabel('Percentage')
    plt.legend()
    plt.grid(True)
    plt.show()

In [20]:
indicator_labels = wellbeing['indicator_label'].unique().tolist()

dropdown = widgets.Dropdown(options=indicator_labels, description='Select Indicator Label:')

def update_plot(indicator_label):
    plot_indicator(indicator_label)

interact(update_plot, indicator_label=dropdown)

interactive(children=(Dropdown(description='Select Indicator Label:', options=('Quality of life', 'How safe yo…

<function __main__.update_plot(indicator_label)>

<div class="usecase-section-heading" id="section-2"><b>2. Economy</b></div>

<div class="usecase-body">This section reviews indicators of employment, poverty, property values and business activity for Greater Melbourne and the Melbourne municipality.</div>

In [24]:
life = datasets['indicators-of-quality-of-life-and-city-services-by-year']
life.head()

,number,indicator_theme,indicator_variable,indicator_descriptor,type,greater_melbourne_value,greater_melbourne_year,greater_melbourne_units,melbourne_municipality_value,melbourne_municipality_year,melbourne_municipality_units
0,5.4,Economy,Full-time Employment,Percentage of persons in fulltime employment,Supporting,50.6319,2014.0,%,58.5449,2014.0,NaN
1,5.7,Economy,Number of New Patents,Number of new patents per 100 000 population p...,Supporting,13.6251,2014.0,/100 000/yr,84.2832,2014.0,NaN
2,6.3,Education,Secondary Education Completion,Percentage of students completing secondary ed...,Core,91.8848,2013.0,%,70.0492,2013.0,NaN
3,7.1,Energy,Residential Electrical Use,Total residential electrical energy use per ca...,Core,NaN,NaN,kWh/yr/capita,1730.8241,2015.0,NaN
4,7.7,Energy,Electrical Interruption Length,Average length of electrical interruptions (in...,Supporting,1.6329,2010.0,hours,1.8827,2010.0,NaN


In [26]:
life[life['indicator_theme'] == "Economy"].sort_values(by='number')

,number,indicator_theme,indicator_variable,indicator_descriptor,type,greater_melbourne_value,greater_melbourne_year,greater_melbourne_units,melbourne_municipality_value,melbourne_municipality_year,melbourne_municipality_units
12,5.1,Economy,Unemployment Rate,City's unemployment rate,Core,6.6497,2014.0,%,5.0346,2014.0,NaN
70,5.2,Economy,Commercial and Industrial Property Values,Assessed value of commercial and industrial pr...,Core,NaN,NaN,%,47.5365,2015.0,NaN
24,5.3,Economy,Population Living in Poverty,Percentage of city population living in poverty,Core,8.2000,2014.0,%,11.0223,2014.0,NaN
0,5.4,Economy,Full-time Employment,Percentage of persons in fulltime employment,Supporting,50.6319,2014.0,%,58.5449,2014.0,NaN
13,5.5,Economy,Youth Unemployment Rate,Youth unemployment rate,Supporting,11.0454,2011.0,%,7.8182,2011.0,NaN
49,5.6,Economy,Number of Businesses,Number of businesses per 100 000 population,Supporting,3584.4199,2014.0,/100 000,11500.1596,2014.0,NaN
1,5.7,Economy,Number of New Patents,Number of new patents per 100 000 population p...,Supporting,13.6251,2014.0,/100 000/yr,84.2832,2014.0,NaN


- The unemployment rate in the city was 6.6497% in 2014, with a higher rate of 11.0223% in the Melbourne municipality. This suggests a potential disparity in employment opportunities between the broader Greater Melbourne area and the specific municipality.
- Commercial and industrial property values experienced a significant increase of 47.5365% in 2015 in the Melbourne municipality. This substantial rise could indicate a robust economic environment or increased investment in commercial and industrial sectors within the municipality.
- The percentage of the population living in poverty was 8.2% in 2014 for Greater Melbourne, whereas it was higher at 11.0223% in the Melbourne municipality. This indicates a need for targeted poverty alleviation measures within the municipality.
- The percentage of persons in full-time employment was 50.6319% in 2014 for Greater Melbourne and increased to 58.5449% in the Melbourne municipality. This suggests a higher concentration of full-time employment opportunities within the municipality compared to the broader metropolitan area.
- The youth unemployment rate was notably higher at 11.0454% in 2011 compared to the overall unemployment rate, indicating potential challenges in youth employment within the region.
- The number of businesses per 100,000 population was substantially higher in the Melbourne municipality compared to Greater Melbourne, indicating a concentration of business activity within the municipality.
- The number of new patents per 100,000 population per year was relatively low in both Greater Melbourne and the Melbourne municipality, suggesting potential areas for fostering innovation and research within the region.

<div class="usecase-body"><b>How this can be used:</b> Researchers and community organisations can use this employment and poverty comparison to target support programs geographically. Local businesses and workforce development agencies can reference the innovation and property-value figures when deciding where to invest or expand. Journalists and advocacy groups can use the disparity between Greater Melbourne and the municipality as a starting point for deeper reporting or campaigns.</div>

<div class="usecase-section-heading" id="section-3"><b>3. Education</b></div>

<div class="usecase-body">This section reviews indicators of school enrolment, completion rates and higher education attainment for Greater Melbourne and the Melbourne municipality.</div>

In [32]:
life[life['indicator_theme'] == "Education"].sort_values(by='number')

,number,indicator_theme,indicator_variable,indicator_descriptor,type,greater_melbourne_value,greater_melbourne_year,greater_melbourne_units,melbourne_municipality_value,melbourne_municipality_year,melbourne_municipality_units
79,6.1,Education,Female Student Enrolment,Percentage of female school-aged population en...,Core,100.0156,2013.0,%,NaN,NaN,NaN
14,6.2,Education,Primary Education Completion,Percentage of students completing primary educ...,Core,102.0663,2013.0,%,NaN,NaN,NaN
2,6.3,Education,Secondary Education Completion,Percentage of students completing secondary ed...,Core,91.8848,2013.0,%,70.0492,2013.0,NaN
55,6.4,Education,Student/Teacher Ratio,Primary education student/teacher ratio,Core,15.1977,2013.0,NaN,14.7139,2013.0,NaN
80,6.5,Education,Male Student Enrolment,Percentage of male school-aged population enro...,Supporting,99.8322,2013.0,%,NaN,NaN,NaN
50,6.6,Education,Total Student Enrolment,Percentage of school-aged population enrolled ...,Supporting,99.9218,2013.0,%,NaN,NaN,NaN
51,6.7,Education,Higher Education Degrees,Number of higher education degrees per 100 000...,Supporting,26388.0731,2014.0,/100 000,46630.7167,2014.0,NaN


- Female student enrolment exceeded 100% in 2013 in Greater Melbourne, indicating potential discrepancies or data anomalies.
- Primary education completion rate was 102.07% in 2013, suggesting possible overestimation or exceptional circumstances.
- Secondary education completion rate was 91.88% in Greater Melbourne and 70.05% in the Melbourne municipality in 2013, showing differences in completion rates.
- The student/teacher ratio was approximately 15 in primary education and slightly lower in the Melbourne municipality in 2013.
- Male student enrolment was around 99.83% in 2013, indicating near parity with female enrolment.
- Total student enrolment was approximately 99.92% in 2013, suggesting high enrolment rates.
- Higher education degrees were more prevalent in the Melbourne municipality, with 46,630.72 per 100,000 population compared to 26,388.07 in Greater Melbourne in 2014.

<div class="usecase-body"><b>How this can be used:</b> School administrators and education researchers can use the completion-rate and student/teacher ratio figures to identify where additional support is most needed. Policymakers designing curriculum or funding allocations can reference the enrolment gender-parity data. Community tutoring groups and parents comparing school-age outcomes across regions can use these indicators as a starting benchmark.</div>

<div class="usecase-section-heading" id="section-4"><b>4. Energy</b></div>

<div class="usecase-body">This section reviews electricity service reliability indicators for Greater Melbourne and the Melbourne municipality.</div>

In [37]:
life[life['indicator_theme'] == "Energy"].sort_values(by='number')

,number,indicator_theme,indicator_variable,indicator_descriptor,type,greater_melbourne_value,greater_melbourne_year,greater_melbourne_units,melbourne_municipality_value,melbourne_municipality_year,melbourne_municipality_units
3,7.1,Energy,Residential Electrical Use,Total residential electrical energy use per ca...,Core,NaN,NaN,kWh/yr/capita,1730.8241,2015.0,NaN
43,7.2,Energy,Authorized Electrical Service,Percentage of city population with authorized ...,Core,100.0000,2013.0,%,100.0000,2013.0,NaN
36,7.3,Energy,Public Building Electricity Use,Energy (electricity) consumption of public bui...,Core,NaN,NaN,kWh/m²,360.7616,2014.0,NaN
77,7.4,Energy,Electricity from Renewable Sources,Percentage of total energy derived from renewa...,Core,NaN,NaN,%,12.3500,2014.0,NaN
35,7.5,Energy,Total Electrical Use,Total electrical energy use per capita (kWh/year),Supporting,NaN,NaN,kWh/yr/capita,27083.7691,2014.0,NaN
25,7.6,Energy,Electrical Interruptions,Average number of electrical interruptions per...,Supporting,1.3069,2010.0,/customer/yr,0.2810,2010.0,NaN
4,7.7,Energy,Electrical Interruption Length,Average length of electrical interruptions (in...,Supporting,1.6329,2010.0,hours,1.8827,2010.0,NaN


- The average number of electrical interruptions per customer per year was 1.31 in Greater Melbourne in 2010 and 0.28 in the Melbourne municipality, indicating a difference in service reliability.
- The average length of electrical interruptions was 1.63 hours in Greater Melbourne in 2010 and 1.88 hours in the Melbourne municipality, suggesting slight variations in outage durations.

<div class="usecase-body"><b>How this can be used:</b> Utility planners and infrastructure investors can use these reliability figures to prioritise maintenance and upgrade decisions. Residents and businesses assessing where to locate can reference outage frequency and duration as a proxy for grid resilience. Researchers studying energy infrastructure can use the regional comparison as a baseline for further analysis.</div>

<div class="usecase-section-heading" id="section-5"><b>5. Environment</b></div>

<div class="usecase-body">This section reviews air quality indicators for Greater Melbourne and the Melbourne municipality.</div>

In [42]:
life[life['indicator_theme'] == "Environment"].sort_values(by='number')

,number,indicator_theme,indicator_variable,indicator_descriptor,type,greater_melbourne_value,greater_melbourne_year,greater_melbourne_units,melbourne_municipality_value,melbourne_municipality_year,melbourne_municipality_units
15,8.1,Environment,PM2.5 Concentration,Fine Particulate Matter (PM2.5) concentration,Core,7.4000,2014.0,?g/m³,7.4000,2014.0,NaN
81,8.2,Environment,PM10 Concentration,Particulate Matter (PM10) concentration,Core,17.4000,2014.0,?g/m³,17.4000,2014.0,NaN
82,8.3,Environment,GHG Emissions,Greenhouse gas emissions measured in tonnes pe...,Core,NaN,NaN,t/capita,42.7262,2014.0,NaN
91,8.4,Environment,NO2 Concentration,NO? (nitrogen dioxide) concentration,Supporting,15.8693,2014.0,?g/m³,15.8693,2014.0,NaN
16,8.5,Environment,SO2 Concentration,SO? (sulphur dioxide) concentration,Supporting,2.1014,2014.0,?g/m³,2.1014,2014.0,NaN
83,8.6,Environment,O3 Concentration,O? (ozone) concentration,Supporting,34.3945,2014.0,?g/m³,34.3945,2014.0,NaN
84,8.7,Environment,Noise Pollution,Noise pollution,Supporting,NaN,NaN,%,NaN,NaN,NaN
37,8.8,Environment,Change in Native Species,Percentage change in number of native species,Supporting,NaN,NaN,%,NaN,NaN,NaN


- Air quality indicators like PM2.5, PM10, NO2, SO2, and O3 concentrations were consistent between Greater Melbourne and its municipalities in 2014, suggesting uniform environmental conditions.

<div class="usecase-body"><b>How this can be used:</b> Environmental researchers and public health analysts can use these air quality figures to study links between pollutant levels and health outcomes. Urban planners and community environmental groups can reference this data when advocating for green infrastructure or sustainable transport initiatives.</div>

<div class="usecase-section-heading" id="section-6"><b>6. Finance</b></div>

<div class="usecase-body">This section reviews municipal financial indicators, including debt, revenue and tax collection, for Greater Melbourne and the Melbourne municipality.</div>

In [47]:
life[life['indicator_theme'] == "Finance"].sort_values(by='number')

,number,indicator_theme,indicator_variable,indicator_descriptor,type,greater_melbourne_value,greater_melbourne_year,greater_melbourne_units,melbourne_municipality_value,melbourne_municipality_year,melbourne_municipality_units
17,9.1,Finance,Debt Service Ratio,Debt service ratio (debt service expenditure a...,Core,1.0252,2015.0,%,0.0000,2015.0,NaN
56,9.2,Finance,Capital Spending,Capital spending as a percentage of total expe...,Supporting,20.2772,2015.0,%,17.9328,2015.0,NaN
92,9.3,Finance,Own-Source Revenues,Own-source revenue as a percentage of total re...,Supporting,75.1170,2015.0,%,86.5048,2015.0,NaN
85,9.4,Finance,Taxes Collected,Tax collected as percentage of tax billed,Supporting,99.6555,2015.0,%,100.9082,2015.0,NaN


- The debt service ratio differs notably between Greater Melbourne and the Melbourne municipality in 2015, indicating potentially divergent financial management approaches or debt burdens across regions.
- While both areas allocated a significant portion of expenditure to capital spending in 2015, the Melbourne municipality's slightly lower percentage suggests a nuanced difference in investment priorities or fiscal strategies.
- Greater Melbourne relied less on own-source revenue compared to the Melbourne municipality in 2015, indicating potential differences in revenue diversification strategies or economic structures between the two areas.
- Both areas demonstrated high tax collection efficiency, but the Melbourne municipality's slightly higher percentage suggests a slightly more effective tax administration system or compliance rate.

<div class="usecase-body"><b>How this can be used:</b> Financial analysts and other municipal councils can use these ratios to benchmark fiscal management practices against Melbourne. Researchers studying municipal finance can reference the revenue diversification and tax-efficiency figures. Journalists reporting on public spending can use the debt-service and capital-spending comparison as a starting point for further investigation.</div>

<div class="usecase-section-heading" id="section-7"><b>7. Fire and Emergency Response</b></div>

<div class="usecase-body">This section reviews fire-related safety indicators for Greater Melbourne and the Melbourne municipality.</div>

In [52]:
life[life['indicator_theme'] == "Fire and Emergency Response"].sort_values(by='number')

,number,indicator_theme,indicator_variable,indicator_descriptor,type,greater_melbourne_value,greater_melbourne_year,greater_melbourne_units,melbourne_municipality_value,melbourne_municipality_year,melbourne_municipality_units
44,10.1,Fire and Emergency Response,Number of Firefighters,Number of firefighters per 100 000 population,Core,NaN,NaN,/100 000,278.2165,2014.0,NaN
78,10.2,Fire and Emergency Response,Number of Fire-Related Deaths,Number of fire related deaths per 100 000 popu...,Core,0.2116,2012.0,/100 000/yr,0.0000,2012.0,NaN
26,10.3,Fire and Emergency Response,Natural Disaster-Related Deaths,Number of natural disaster related deaths per ...,Core,0.0000,2012.0,/100 000/yr,0.0000,2012.0,NaN
5,10.4,Fire and Emergency Response,Volunteer Firefighters,Number of volunteer and part-time firefighters...,Supporting,NaN,NaN,/100 000,0.0000,2013.0,NaN
57,10.5,Fire and Emergency Response,Emergency Response Time,Response time for emergency response services ...,Supporting,NaN,NaN,min,NaN,NaN,NaN
58,10.6,Fire and Emergency Response,Fire Response Time,Response time for fire department from initial...,Supporting,NaN,NaN,min,NaN,NaN,NaN


- The Melbourne municipality reported a lower rate of fire-related deaths compared to Greater Melbourne in 2012. This indicates potentially more effective fire safety measures or emergency response systems in the municipality.

<div class="usecase-body"><b>How this can be used:</b> Emergency planners and insurers can use the fire-related death rate as one input when assessing regional risk. Community safety groups and researchers in public safety can reference this figure alongside other datasets, noting the limited data coverage currently available for this theme.</div>

<div class="usecase-section-heading" id="section-8"><b>8. Governance</b></div>

<div class="usecase-body">This section reviews civic participation, representation and transparency indicators for Greater Melbourne and the Melbourne municipality.</div>

In [58]:
life[life['indicator_theme'] == "Governance"].sort_values(by='number')

,number,indicator_theme,indicator_variable,indicator_descriptor,type,greater_melbourne_value,greater_melbourne_year,greater_melbourne_units,melbourne_municipality_value,melbourne_municipality_year,melbourne_municipality_units
59,11.1,Governance,Voter Participation Rate,Voter participation in last municipal election...,Core,69.0133,2012.0,%,60.0098,2012.0,NaN
6,11.2,Governance,Women Elected to Office,Women as a percentage of total elected to city...,Core,36.8771,2012.0,%,36.3636,2015.0,NaN
71,11.3,Governance,Women Employed in City Government,Percentage of women employed in the city gover...,Supporting,NaN,NaN,%,38.8065,2013.0,NaN
38,11.4,Governance,Number of Corruption Convictions,Number of convictions for corruption and / or ...,Supporting,NaN,NaN,/100 000,NaN,NaN,NaN
86,11.5,Governance,Citizen Representation,Citizens' representation: number of local offi...,Supporting,7.0783,2012.0,/100 000,9.0011,2014.0,NaN
27,11.6,Governance,Number of Registered Voters,Number of registered voters as a percentage of...,Supporting,87.6410,2012.0,%,129.6933,2012.0,NaN


- Voter participation rates were higher in Greater Melbourne compared to the Melbourne municipality in 2012, indicating varying levels of civic engagement across regions. This suggests a need for targeted initiatives to encourage voter participation, especially in areas with lower turnout.
- Both Greater Melbourne and the Melbourne municipality demonstrated relatively similar levels of gender representation in elected offices in 2012. However, efforts to improve gender inclusivity in city government employment were evident in the Melbourne municipality in 2013. This highlights ongoing endeavours to enhance gender diversity in municipal governance roles.
- Data on corruption convictions were not available for Greater Melbourne, suggesting a potential gap in transparency measures or reporting standards. Enhancing transparency and anti-corruption efforts could bolster public trust and accountability in governance.
- The number of local officials per 100,000 citizens varied between Greater Melbourne and the Melbourne municipality in 2012, indicating differences in citizen representation structures. Understanding these variations can inform discussions on optimising governance structures to better serve the population's needs.
- The percentage of registered voters was notably higher in the Melbourne municipality compared to Greater Melbourne in 2012. This discrepancy raises questions about voter registration practices and data accuracy, emphasising the importance of ensuring inclusive and accurate voter registration processes.

<div class="usecase-body"><b>How this can be used:</b> Civic engagement organisations can use the voter turnout and registration figures to target outreach efforts. Election researchers and transparency/anti-corruption watchdogs can reference the gender-representation and corruption-conviction data. Students and academics studying municipal governance can use these indicators as a case study in comparative civic participation.</div>

<div class="usecase-section-heading" id="section-9"><b>9. Health</b></div>

<div class="usecase-body">This section reviews healthcare access, staffing and outcome indicators for Greater Melbourne and the Melbourne municipality.</div>

In [64]:
life[life['indicator_theme'] == "Health"].sort_values(by='number')

,number,indicator_theme,indicator_variable,indicator_descriptor,type,greater_melbourne_value,greater_melbourne_year,greater_melbourne_units,melbourne_municipality_value,melbourne_municipality_year,melbourne_municipality_units
28,12.1,Health,Life Expectancy,Average life expectancy,Core,83.5000,2013.0,years,83.5000,2013.0,NaN
18,12.2,Health,In-Patient Hospital Beds,Number of in-patient hospital beds per 100 000...,Core,349.7489,2014.0,/100 000,2114.4397,2012.0,NaN
72,12.3,Health,Number of Physicians,Number of physicians per 100 000 population,Core,369.9119,2014.0,/100 000,2514.5041,2014.0,NaN
93,12.4,Health,Under Age Five Mortality Rate,Under age five mortality per 1 000 live births,Core,3.3000,2013.0,/1000 live births,5.4000,2013.0,NaN
29,12.5,Health,Number of Nurses/Midwives,Number of nursing and midwifery personnel per ...,Supporting,979.5966,2014.0,/100 000,5024.5894,2014.0,NaN
7,12.6,Health,Number of Mental Health Practitioners,Number of mental health practitioners per 100 ...,Supporting,188.5469,2014.0,/100 000,1090.9359,2014.0,NaN
60,12.7,Health,Suicide Rate,Suicide rate per 100 000 population,Supporting,8.3951,2012.0,/100 000,8.5387,2012.0,NaN


- The average life expectancy was consistent at 83.5 years for both Greater Melbourne and the Melbourne municipality in 2013, indicating comparable health outcomes across regions.
- The number of in-patient hospital beds per 100,000 population was significantly higher in the Melbourne municipality (2114.44 beds) compared to Greater Melbourne (349.75 beds) in 2014. This suggests potentially greater healthcare infrastructure and capacity in the municipality.
- Greater Melbourne had 369.91 physicians per 100,000 population in 2014, while the Melbourne municipality had a notably higher number at 2514.50 physicians. This indicates differences in access to healthcare professionals between the two regions.
- The under-five mortality rate was lower in Greater Melbourne (3.3 per 1000 live births) compared to the Melbourne municipality (5.4 per 1000 live births) in 2013, reflecting differences in child healthcare and public health interventions.
- The number of nursing and midwifery personnel per 100,000 population was higher in the Melbourne municipality (5024.59 personnel) compared to Greater Melbourne (979.60 personnel) in 2014, indicating potentially greater healthcare staffing levels in the municipality.
- The number of mental health practitioners per 100,000 population was higher in the Melbourne municipality (1090.94 practitioners) compared to Greater Melbourne (188.55 practitioners) in 2014, suggesting potentially greater access to mental health services in the municipality.
- The suicide rate was comparable between Greater Melbourne (8.40 per 100,000 population) and the Melbourne municipality (8.54 per 100,000 population) in 2012, indicating consistent mental health challenges across regions.

<div class="usecase-body"><b>How this can be used:</b> Healthcare planners and public health researchers can use the staffing and bed-capacity figures to identify where infrastructure investment is most needed. NGOs and outreach programs targeting underserved regions can reference the mental health practitioner and mortality figures. Students and academics can use this regional comparison as a case study in healthcare-access disparities.</div>

<div class="usecase-section-heading" id="section-10"><b>10. Recreation</b></div>

<div class="usecase-body">Use this section to review current public recreation space provision in the Melbourne municipality, and to flag where comparable Greater Melbourne figures are still needed for proper benchmarking..</div>

In [69]:
life[life['indicator_theme'] == "Recreation"].sort_values(by='number')

,number,indicator_theme,indicator_variable,indicator_descriptor,type,greater_melbourne_value,greater_melbourne_year,greater_melbourne_units,melbourne_municipality_value,melbourne_municipality_year,melbourne_municipality_units
94,13.1,Recreation,Indoor Recreation Space,Square metres of public indoor recreation spac...,Supporting,NaN,NaN,m²/capita,8.5415,2012.0,NaN
8,13.2,Recreation,Outdoor Recreation Space,Square metres of public outdoor recreation spa...,Supporting,NaN,NaN,m²/capita,14.6141,2012.0,NaN


- The Melbourne municipality had 8.5415 m² per capita of public indoor recreation space in 2012. No equivalent figure was available for Greater Melbourne.
- The Melbourne municipality had 14.6141 m² per capita of public outdoor recreation space in 2012 — over 70% more space than indoor recreation. No equivalent figure was available for Greater Melbourne.

<div class="usecase-body"><b>How this can be used:</b> Urban planners and recreation providers can use the indoor/outdoor space-per-capita figures to assess where facilities may need expansion. Community groups advocating for more recreation space, and researchers studying liveability, can reference this data — noting that equivalent Greater Melbourne figures are not yet available for direct comparison.</div>

<div class="usecase-section-heading" id="section-11"><b>11. Safety</b></div>

<div class="usecase-body">This section reviews crime and policing indicators for Greater Melbourne and the Melbourne municipality.</div>

In [75]:
life[life['indicator_theme'] == "Safety"].sort_values(by='number')

,number,indicator_theme,indicator_variable,indicator_descriptor,type,greater_melbourne_value,greater_melbourne_year,greater_melbourne_units,melbourne_municipality_value,melbourne_municipality_year,melbourne_municipality_units
87,14.1,Safety,Number of Police Officers,Number of police officers per 100 000 population,Core,132.5499,2014.0,/100 000/yr,253.8398,2014.0,NaN
73,14.2,Safety,Number of Homicides,Number of homicides per 100 000 population,Core,2.1395,2014.0,/100 000/yr,8.1828,2014.0,NaN
9,14.3,Safety,Number of Property Crimes,Crimes against property per 100 000 population,Supporting,4253.2894,2014.0,/100 000/yr,10721.9717,2014.0,NaN
39,14.4,Safety,Police Response Time,Response time for police department from initi...,Supporting,NaN,NaN,min,NaN,NaN,NaN
40,14.5,Safety,Number of Violent Crimes,Violent crime rate per 100 000 population,Supporting,1040.1259,2014.0,/100 000/yr,2849.2640,2014.0,NaN


- The number of police officers per 100,000 population was higher in the Melbourne municipality (253.84 officers) compared to Greater Melbourne (132.55 officers) in 2014. This indicates potentially greater law enforcement resources and coverage in the municipality.
- The homicide rate per 100,000 population was higher in the Melbourne municipality (8.18 homicides) compared to Greater Melbourne (2.14 homicides) in 2014. This suggests potentially differing levels of crime and safety concerns between the two regions.
- The rate of crimes against property per 100,000 population was higher in the Melbourne municipality (10721.97 crimes) compared to Greater Melbourne (4253.29 crimes) in 2014. This indicates potentially higher incidences of property-related offenses in the municipality.
- The rate of violent crimes per 100,000 population was higher in the Melbourne municipality (2849.26 crimes) compared to Greater Melbourne (1040.13 crimes) in 2014. This suggests potentially greater challenges with violent crime prevention and law enforcement in the municipality.

<div class="usecase-body"><b>How this can be used:</b> Police and crime-prevention agencies can use these rates to allocate resources across regions. Community safety groups, insurers and journalists can reference the crime-rate comparison, and researchers can use the dataset to study the relationship between officer coverage and crime outcomes.</div>

<div class="usecase-section-heading" id="section-12"><b>12. Shelter</b></div>

<div class="usecase-body">This section reviews housing and homelessness indicators for Greater Melbourne and the Melbourne municipality.</div>

In [80]:
life[life['indicator_theme'] == "Shelter"].sort_values(by='number')

,number,indicator_theme,indicator_variable,indicator_descriptor,type,greater_melbourne_value,greater_melbourne_year,greater_melbourne_units,melbourne_municipality_value,melbourne_municipality_year,melbourne_municipality_units
41,15.1,Shelter,Population Living in Slums,Percentage of city population living in slums,Core,0.4705,2011.0,%,0.6184,2011.0,NaN
42,15.2,Shelter,Number of Homeless,Number of homeless per 100 000 population,Supporting,156.8800,2011.0,/100 000,395.0500,2011.0,NaN
19,15.3,Shelter,Unregistered Households,Percentage of households that exist without re...,Supporting,0.0000,2011.0,%,0.0000,2014.0,NaN


- In 2011, a small percentage of the population lived in slums in both Greater Melbourne (0.47%) and the Melbourne municipality (0.62%). This suggests relatively low incidences of slum living, indicating a generally favourable housing situation.
- In 2011, there were 156.88 homeless individuals per 100,000 population in Greater Melbourne and 395.05 homeless individuals per 100,000 population in the Melbourne municipality. This indicates a higher prevalence of homelessness in the municipality compared to the broader Greater Melbourne area.

<div class="usecase-body"><b>How this can be used:</b> Housing policy researchers and social services can use the homelessness-rate comparison to target support programs. NGOs and community housing providers can reference these figures when applying for funding or planning outreach, and urban planners can use them alongside the Urban Planning section's jobs/housing ratio for a fuller housing picture.</div>

<div class="usecase-section-heading" id="section-13"><b>13. Solid Waste</b></div>

<div class="usecase-body">This section reviews solid waste collection coverage indicators for Greater Melbourne and the Melbourne municipality.</div>

In [85]:
life[life['indicator_theme'] == "Solid Waste"].sort_values(by='number')

,number,indicator_theme,indicator_variable,indicator_descriptor,type,greater_melbourne_value,greater_melbourne_year,greater_melbourne_units,melbourne_municipality_value,melbourne_municipality_year,melbourne_municipality_units
52,16.1,Solid Waste,Hazardous Waste Recycled,Percentage of city's hazardous waste that is r...,Supporting,NaN,NaN,%,NaN,NaN,NaN
61,16.1,Solid Waste,Residential Solid Waste Collection,Percentage of city population with regular sol...,Core,100.0000,2014.0,%,100.0000,2015.0,NaN
62,16.2,Solid Waste,Solid Waste Collected,Total collected municipal solid waste per capita,Core,0.3921,2013.0,t/capita,0.3384,2013.0,NaN
63,16.3,Solid Waste,Solid Waste Recycled,Percentage of the city's solid waste that is r...,Core,43.6805,2013.0,%,20.9427,2013.0,NaN
95,16.4,Solid Waste,Disposal by Landfill,Percentage of the city's solid waste that is d...,Supporting,56.3195,2013.0,%,79.0573,2013.0,NaN
45,16.5,Solid Waste,Disposal by Incinerator,Percentage of the city's solid waste that is d...,Supporting,0.0000,2013.0,%,0.0000,2013.0,NaN
30,16.6,Solid Waste,Disposal by Burning Openly,Percentage of the city's solid waste that is b...,Supporting,0.0000,2013.0,%,0.0000,2013.0,NaN
46,16.7,Solid Waste,Disposal by Open Dump,Percentage of the city's solid waste that is d...,Supporting,0.0000,2013.0,%,0.0000,2013.0,NaN
64,16.8,Solid Waste,Disposal by Other,Percentage of the city's solid waste that is d...,Supporting,0.0000,2013.0,%,0.0000,2013.0,NaN
88,16.9,Solid Waste,Hazardous Waste Generated,Hazardous waste generation per capita,Supporting,NaN,NaN,t/capita,NaN,NaN,NaN


- Both Greater Melbourne and the Melbourne municipality had 100% of their respective populations covered by regular solid waste collection services in 2014 and 2015. This indicates comprehensive waste management infrastructure and services in place to handle residential waste.

<div class="usecase-body"><b>How this can be used:</b> Waste management providers and other councils can use this full-coverage figure as a benchmark for service standards. Sustainability researchers and environmental groups can reference it as a baseline before comparing recycling and landfill-diversion rates once that data becomes available.</div>

<div class="usecase-section-heading" id="section-14"><b>14. Telecommunication and Innovation</b></div>

<div class="usecase-body">This section reviews internet and mobile connectivity indicators for Greater Melbourne and the Melbourne municipality.</div>

In [90]:
life[life['indicator_theme'] == "Telecommunication and Innovation"].sort_values(by='number')

,number,indicator_theme,indicator_variable,indicator_descriptor,type,greater_melbourne_value,greater_melbourne_year,greater_melbourne_units,melbourne_municipality_value,melbourne_municipality_year,melbourne_municipality_units
20,17.1,Telecommunication and Innovation,Internet Connections,Number of internet connections per 100 000 pop...,Core,53631.9513,2014.0,/100 000,53631.9513,2014.0,NaN
89,17.2,Telecommunication and Innovation,Cell Phone Connections,Number of cell phone connections per 100 000 p...,Core,84935.2122,2013.0,/100 000,84935.2122,2013.0,NaN
31,17.3,Telecommunication and Innovation,Landline Connections,Number of landline phone connections per 100 0...,Supporting,NaN,NaN,/100 000,NaN,NaN,NaN


- Both Greater Melbourne and the Melbourne municipality had a high number of internet connections per 100,000 population in 2014, with approximately 53,631.95 connections. This indicates widespread access to the internet, essential for communication, information access, and economic participation.
- Similarly, both Greater Melbourne and the Melbourne municipality had a high number of cell phone connections per 100,000 population in 2013, with approximately 84,935.21 connections. This suggests extensive mobile network coverage and high mobile phone adoption rates, facilitating communication and connectivity.

<div class="usecase-body"><b>How this can be used:</b> Digital inclusion advocates and telecom providers can use the connectivity figures to identify where infrastructure investment could improve access. Economic development agencies and innovation researchers can reference this data when assessing a region's readiness for digital economic participation.</div>

<div class="usecase-section-heading" id="section-15"><b>15. Transportation</b></div>

<div class="usecase-body">This section reviews public transport, active transport and road safety indicators for Greater Melbourne and the Melbourne municipality.</div>

In [96]:
life[life['indicator_theme'] == "Transportation"].sort_values(by='number')

,number,indicator_theme,indicator_variable,indicator_descriptor,type,greater_melbourne_value,greater_melbourne_year,greater_melbourne_units,melbourne_municipality_value,melbourne_municipality_year,melbourne_municipality_units
65,18.1,Transportation,High-Capacity Public Transit,Kilometres of high capacity public transport s...,Core,28.4601,2015.0,km/100 000,102.8714,2015.0,NaN
90,18.2,Transportation,Light Passenger Public Transit,Kilometres of light passenger public transport...,Core,180.3500,2015.0,km/100 000,293.5700,2015.0,NaN
66,18.3,Transportation,Public Transit Trips,Annual number of public transport trips per ca...,Core,81.7152,2014.0,capita/yr,1009.1830,2014.0,NaN
32,18.4,Transportation,Number of Automobiles,Number of personal automobiles per capita,Core,0.5967,2013.0,/capita,0.7167,2013.0,NaN
53,18.5,Transportation,Mode Share,Percentage of commuters using a travel mode to...,Supporting,29.1323,2011.0,%,65.8252,2011.0,NaN
67,18.6,Transportation,Number of 2-Wheeled Vehicles,Number of two-wheel motorized vehicles per capita,Supporting,0.0251,2013.0,/capita,0.0249,2013.0,NaN
21,18.7,Transportation,Bicycle Paths,Kilometres of bicycle paths and lanes per 100 ...,Supporting,98.4824,2014.0,km/100 000,147.4204,2014.0,NaN
96,18.8,Transportation,Transportation Fatalities,Transportation fatalities per 100 000 population,Supporting,2.6239,2013.0,/100 000/yr,1.7191,2013.0,NaN
68,18.9,Transportation,Commercial Air Connectivity,Commercial air connectivity,Supporting,114368.0000,2014.0,/yr,114368.0000,2014.0,NaN


- In 2015, Greater Melbourne had 28.46 kilometres of high-capacity public transport services per 100,000 population, while the Melbourne municipality had a notably higher coverage of 102.87 kilometres. This suggests a more extensive high-capacity public transit network in the municipality, potentially providing better access to mass transit options for residents.
- In 2015, Greater Melbourne had 180.35 kilometres of light passenger public transport services per 100,000 population, while the Melbourne municipality had a higher coverage of 293.57 kilometres. This indicates a more comprehensive coverage of light passenger public transit options in the municipality.
- The annual number of public transport trips per capita was 81.72 in Greater Melbourne and substantially higher at 1009.18 in the Melbourne municipality in 2014. This reflects a higher usage of public transit in the municipality, possibly due to better service coverage and accessibility.
- Both Greater Melbourne and the Melbourne municipality had a relatively low number of personal automobiles per capita in 2013, with 0.5967 and 0.7167 automobiles per capita, respectively. This suggests a relatively low dependence on personal vehicles for transportation in both areas.
- In 2011, a higher percentage of commuters in the Melbourne municipality (65.83%) used a travel mode other than private automobiles, compared to Greater Melbourne (29.13%). This indicates a higher adoption of alternative transportation modes such as public transit, walking, or cycling in the municipality.
- The number of two-wheeled motorised vehicles per capita was relatively low in both Greater Melbourne and the Melbourne municipality in 2013, suggesting a limited prevalence of motorised two-wheeled vehicles compared to other forms of transportation.
- In 2014, the Melbourne municipality had a higher coverage of bicycle paths and lanes per 100,000 population (147.42 kilometres) compared to Greater Melbourne (98.48 kilometres). This suggests potentially better infrastructure for cycling in the municipality, promoting active transportation options.
- The transportation fatalities per 100,000 population were slightly higher in Greater Melbourne (2.62) compared to the Melbourne municipality (1.72) in 2013, indicating potential differences in road safety measures or infrastructure between the two areas.
- Both Greater Melbourne and the Melbourne municipality had the same level of commercial air connectivity, with 114,368 connections in 2014. This suggests similar access to air travel options for residents and businesses across the region.

<div class="usecase-body"><b>How this can be used:</b> Transport planners and urban mobility researchers can use the public-transit coverage figures to identify where network expansion would have the greatest impact. Cycling and pedestrian advocacy groups can reference the bicycle-infrastructure comparison, and businesses evaluating logistics or site location can use the automobile-dependence and transit-usage figures.</div>

<div class="usecase-section-heading" id="section-16"><b>16. Urban Planning</b></div>

<div class="usecase-body">This section reviews tree canopy and land-use indicators for Greater Melbourne and the Melbourne municipality.</div>

In [101]:
life[life['indicator_theme'] == "Urban Planning"].sort_values(by='number')

,number,indicator_theme,indicator_variable,indicator_descriptor,type,greater_melbourne_value,greater_melbourne_year,greater_melbourne_units,melbourne_municipality_value,melbourne_municipality_year,melbourne_municipality_units
33,19.1,Urban Planning,Green Area,Green area (hectares) per 100 000 population,Core,NaN,NaN,ha/100 000,451.9344,2015.0,NaN
97,19.2,Urban Planning,Trees Planted,Annual number of trees planted per 100 000 pop...,Supporting,NaN,NaN,/100 000/yr,2640.4785,2015.0,NaN
22,19.3,Urban Planning,Informal Settlements,Areal size of informal settlements as a percen...,Supporting,0.0000,2014.0,%,0.0000,2015.0,NaN
76,19.4,Urban Planning,Jobs/Housing Ratio,Jobs/housing ratio,Supporting,1.2015,2015.0,NaN,7.6303,2012.0,NaN


- In 2015, the Melbourne municipality planted a significantly higher number of trees per 100,000 population (2,640.48) compared to data availability for Greater Melbourne. This indicates a proactive approach to urban greening and tree canopy expansion within the municipality, which can contribute to improved air quality, biodiversity, and urban aesthetics.
- In 2015, the jobs/housing ratio was 1.2015 in Greater Melbourne and notably higher at 7.6303 in the Melbourne municipality in 2012. A higher ratio indicates a greater concentration of employment opportunities relative to housing units. This disparity may have implications for commuting patterns, urban sprawl, and access to employment opportunities within the municipality.

<div class="usecase-body"><b>How this can be used:</b> Urban planners and developers can use the jobs/housing ratio to assess commuting pressure and housing supply needs. Environmental groups can reference the tree-planting figures when advocating for urban greening, and employment/commuting researchers can use this section alongside the Transportation and Shelter sections for a fuller picture of urban liveability.</div>

<div class="usecase-section-heading" id="section-17"><b>17. Wastewater</b></div>

<div class="usecase-body">This section reviews wastewater collection and treatment coverage indicators for Greater Melbourne and the Melbourne municipality.</div>

In [107]:
life[life['indicator_theme'] == "Wastewater"].sort_values(by='number')

,number,indicator_theme,indicator_variable,indicator_descriptor,type,greater_melbourne_value,greater_melbourne_year,greater_melbourne_units,melbourne_municipality_value,melbourne_municipality_year,melbourne_municipality_units
47,20.1,Wastewater,Wastewater Collection,Percentage of city population served by wastew...,Core,100.0,2015.0,%,100.0,2015.0,NaN
98,20.2,Wastewater,No Wastewater Treatment,Percentage of the city's wastewater that has r...,Core,0.0,2014.0,%,0.0,2015.0,NaN
10,20.3,Wastewater,Primary Wastewater Treatment,Percentage of the city's wastewater receiving ...,Core,100.0,2014.0,%,100.0,2015.0,NaN
11,20.4,Wastewater,Secondary Wastewater Treatment,Percentage of the city's wastewater receiving ...,Core,100.0,2014.0,%,100.0,2015.0,NaN
99,20.5,Wastewater,Tertiary Wastewater Treatment,Percentage of the city's wastewater receiving ...,Core,NaN,NaN,%,100.0,2015.0,NaN


- Both Greater Melbourne and the Melbourne municipality achieved 100% coverage of wastewater collection in 2015. This indicates comprehensive infrastructure for collecting wastewater from households and businesses, contributing to public health and environmental protection by preventing contamination of water sources.
- Both areas reported 0% of wastewater left untreated in 2014 and 2015. This signifies that all wastewater generated within Greater Melbourne and the Melbourne municipality underwent some form of treatment before discharge, aligning with environmental regulations and standards.
- In 2014, 100% of the city's wastewater received primary treatment, indicating the removal of solid materials and debris. This initial treatment step is essential for reducing the organic load and improving the quality of wastewater before further processing.
- Similarly, in 2014, 100% of the city's wastewater underwent secondary treatment, which involves biological processes to further reduce pollutants and organic matter. This step is crucial for achieving higher water quality standards and minimizing environmental impacts.

<div class="usecase-body"><b>How this can be used:</b> Water utility engineers and environmental regulators can use these treatment-coverage figures as a compliance and performance benchmark. Sustainability researchers and other municipalities can reference this data when planning wastewater infrastructure or exploring water-recycling opportunities.</div>

<div class="usecase-section-heading" id="section-18"><b>18. Water and Sanitation</b></div>

<div class="usecase-body">This section reviews potable water access, consumption and sanitation indicators for Greater Melbourne and the Melbourne municipality.</div>

In [113]:
life[life['indicator_theme'] == "Water and Sanitation"].sort_values(by='number')

,number,indicator_theme,indicator_variable,indicator_descriptor,type,greater_melbourne_value,greater_melbourne_year,greater_melbourne_units,melbourne_municipality_value,melbourne_municipality_year,melbourne_municipality_units
54,21.1,Water and Sanitation,Potable Water Service,Percentage of city population with potable wat...,Core,100.0000,2014.0,%,100.0000,2014.0,NaN
34,21.2,Water and Sanitation,Improved Water Access,Percentage of city population with sustainable...,Core,100.0000,2014.0,%,100.0000,2014.0,NaN
74,21.3,Water and Sanitation,Improved Sanitation Access,Percentage of population with access to improv...,Core,100.0000,2014.0,%,100.0000,2015.0,NaN
23,21.4,Water and Sanitation,Domestic Water Consumption,Total domestic water consumption per capita (l...,Core,164.5041,2014.0,L/d/capita,147.2080,2014.0,NaN
48,21.5,Water and Sanitation,Total Water Consumption,Total water consumption per capita (litres/day),Supporting,226.6007,2014.0,L/d/capita,502.0710,2014.0,NaN
69,21.6,Water and Sanitation,Water Service Interruptions,Average annual hours of water service interrup...,Supporting,NaN,NaN,hrs/yr/household,0.0929,2015.0,NaN
75,21.7,Water and Sanitation,Water Loss,Percentage of water loss (unaccounted for water),Supporting,10.6329,2014.0,%,10.6329,2014.0,NaN


- Both Greater Melbourne and the Melbourne municipality achieved 100% coverage of potable water service in 2014. This indicates that all residents have access to safe and drinkable water, which is essential for public health and well-being.
- Similarly, 100% of the city population had improved water access in 2014, ensuring sustainable and reliable access to clean water sources. This reflects efforts to provide adequate infrastructure and services for water supply across both areas.
- In 2014, both areas reported 100% access to improved sanitation facilities, such as toilets and sewage systems. This indicates widespread access to hygienic sanitation facilities, contributing to public health and environmental sanitation.
- Greater Melbourne had a domestic water consumption rate of 164.5041 litres per day per capita in 2014, while the Melbourne municipality reported a slightly lower rate of 147.2080 litres per day per capita. Variations in consumption rates may reflect differences in household sizes, lifestyles, and water use behaviours between urban and suburban areas.
- The total water consumption per capita was notably higher in Greater Melbourne (226.6007 litres per day per capita) compared to the Melbourne municipality (502.0710 litres per day per capita) in 2014. This disparity may be attributed to factors such as industrial water use, commercial activities, and irrigation practices, which could be more prevalent in the urbanised region of Greater Melbourne.
- Both Greater Melbourne and the Melbourne municipality reported a water loss percentage of 10.6329% in 2014. Water loss, or unaccounted-for water, can occur due to leaks, theft, or inefficient distribution systems. Efforts to minimise water loss are crucial for conserving water resources and ensuring sustainable water management practices.

<div class="usecase-body"><b>How this can be used:</b> Water utilities and conservation groups can use the consumption and water-loss figures to prioritise distribution-infrastructure upgrades. Public health researchers can reference the sanitation-access figures, and residents or consumer-advocacy groups can use the per-capita consumption comparison to benchmark household water use.</div>

<div class="usecase-unnumbered-heading"><b>Conclusion</b></div>

<div class="usecase-sub-section-heading"><b>What we achieved in this analysis</b></div>

<div class="usecase-body">Following the Scenario's goal of reviewing wellbeing and quality-of-life indicators to guide council funding decisions, we combined two Future Melbourne datasets, forecast future values for each wellbeing indicator using an ARIMA model with printed accuracy metrics, and compared 17 quality-of-life themes between Greater Melbourne and the Melbourne municipality to surface where the two regions diverge.</div>

<div class="usecase-sub-section-heading"><b>What we learned</b></div>

Bringing together the recommendations raised across all 17 quality-of-life themes, the city council should prioritise the following:

- **Economy:** Focus on targeted employment initiatives, youth empowerment, poverty alleviation and small business promotion to address disparities in employment and poverty between the Melbourne municipality and Greater Melbourne.
- **Education:** Implement support programs for students in the municipality, particularly to lift secondary completion rates, alongside a review of teacher allocations and continued promotion of gender equity in education.
- **Energy:** Prioritise infrastructure maintenance and upgrades to reduce electrical interruption frequency and duration, particularly in areas with longer outages.
- **Environment:** Continue monitoring air quality and collaborate with stakeholders on sustainable transport and green infrastructure to preserve current air quality standards.
- **Finance:** Analyse the debt-service, capital-spending and revenue-diversification differences between regions to improve financial management practices and tax administration effectiveness.
- **Fire and Emergency Response:** Expand fire and emergency response data collection to enable a fuller, more reliable assessment of safety outcomes across the region.
- **Governance:** Investigate the causes of lower voter turnout and registration discrepancies, build on progress in gender inclusivity in government employment, and close the transparency gap around corruption-conviction reporting.
- **Health:** Invest in healthcare infrastructure and staffing in areas with lower capacity, and continue targeted mental health and child healthcare initiatives.
- **Recreation:** Prioritise collecting equivalent Greater Melbourne recreation-space figures to properly benchmark provision and assess whether current levels are keeping pace with population growth.
- **Safety:** Increase police presence, enhance community policing, and implement crime-prevention initiatives tailored to each area's specific needs.
- **Shelter:** Expand affordable housing programs, supportive services and homelessness-prevention strategies, particularly within the Melbourne municipality.
- **Solid Waste:** With full collection coverage already achieved, shift focus toward improving recycling rates and landfill diversion.
- **Telecommunication and Innovation:** With connectivity already high, focus on ensuring equitable digital access and economic participation for underserved communities.
- **Transportation:** Expand public transit infrastructure and encourage alternative transport modes, particularly in Greater Melbourne where coverage is comparatively lower, while continuing to improve road safety.
- **Urban Planning:** Continue supporting tree-planting and greening programs while monitoring the jobs/housing ratio to manage commuting pressures.
- **Wastewater:** Maintain full treatment coverage as the population grows, and explore water-recycling and resource-recovery opportunities.
- **Water and Sanitation:** Invest in distribution-infrastructure upgrades to reduce the current water loss rate and support long-term water security, despite already-comprehensive access.
- **Data-driven Decision Making:** Across all themes, emphasise collaboration with community groups, businesses, healthcare providers and educational institutions, using this kind of comparative indicator data to guide equitable, evidence-based development across the Melbourne municipality.

<div class="usecase-sub-section-heading"><b>Further opportunities</b></div>

<div class="usecase-body">Recreation-theme indicators were not returned by the current dataset filter and should be revisited when that data becomes available. Several themes (Governance, Telecommunication and Innovation, Urban Planning, Wastewater, and Water and Sanitation) also had thinner source commentary than others and would benefit from deeper investigation once more recent data years are published. Future iterations could also extend the ARIMA forecasting approach used for the wellbeing dataset to the quality-of-life indicators, to give the council forward-looking projections across all 17 themes rather than historical comparisons alone.</div>